Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmissionParameter.java

```diff
@@ -327,6 +327,14 @@ private Integer parseIntegerSafety(@Nullable Object maybeInteger) {
         return messageList;
     }
 
+    @JsonIgnore
+    public Map<String, Object> updateJobConfig(String key, Object value) {
```

Comment:
I prefer to keep this file minimal changes. We can do it at calling side as below:
```
submitModel.submissionParameter.getJobConfig().put(SparkSubmissionParameter.Conf, ...)
```